src/collectors/snmpinterfacepoll/snmpinterfacepoll.py

```diff
@@ -91,17 +95,21 @@ def publish_metrics(self, gaugesList, countersList, timestamp):
                     value=metricValue,
                     oldUnit='byte',
                     newUnit=unit)
-                    metric = Metric(
-                            metricPathNewUnit,
-                            self.derivative(
-                                            metricPathNewUnit,
-                                            metricValueNewUnit,
-                                            COUNTER_MAX_64BIT
-                                            ),
-                            raw_value=None,
-                            timestamp=timestamp
-                            )
-                    self.publish_metric(metric)
+                    # Fixes devices that return no value
```

Comment:
This comment doesn't really provide any value to the reader who doesn't know what the code looked like before to know what was "fixed".